02.calendar/cal.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+
+const makeCalHeader = (year, month) => {
+  return `      ${month + 1}月 ${year}
+日 月 火 水 木 金 土`;
+};
+
+const makeCalBody = (year, month) => {
+  let content = "";
+  let saturdayCount = 0;
+  const firstDate = new Date(year, month, 1);
+  const lastDate = new Date(year, month + 1, 0);
+
+  content += "   ".repeat(firstDate.getDay());
+
+  for (
+    let date = new Date(firstDate.getTime());
+    date <= lastDate;
+    date.setDate(date.getDate() + 1)
+  ) {
+    let displayDate = date.getDate().toString();
+
+    if (displayDate.length === 1) {
+      displayDate = ` ${displayDate}`;
+    }
+
+    let delimiter = "";
+    if (date.getDay() === 6) {
+      delimiter = "\n";
+      saturdayCount++;
+    } else {
+      delimiter = " ";
+    }
+
+    displayDate += delimiter;
+    content += displayDate;
```

Comment:
以下のように書いた方が読みやすいと思います。

```suggestion
    content += `${displayDate}${delimiter}`;
```